src/thor/dao/config.py

```diff
@@ -1,7 +1,5 @@
-### config.py
-
-# Scheme: "postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>" pragma: allowlist secret
-
+# Scheme: "postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>" pragma: allowlist secret
```

Comment:
Might be worth building a new `.secrets-baseline` file so you don't have to use a bunch of the `pragma: allowlist secret` in the file. You can build this using detect-secrets

These are instructions for 0.13.1 but this repo might be using something different...
```
detect-secrets scan --update .secrets.baseline
detect-secrets audit .secrets.baseline
```